src/sections/Meshery/meshery.style.js

```diff
@@ -91,12 +91,19 @@ const MesheryWrapper = styled.section`
             .description {
                 margin-top: 2rem;
 
+                
                 .meshery-hero-img {
+                    min-width:30rem;
                     text-align: center;
                     padding: 1rem 1rem 1rem 1rem;
                 }
-
+                .meshery-hero-img {
```

Comment:
Am I reading this wrong or did you repeat the same class twice? If so, is there a special need to do so?